core/src/main/scala/kafka/server/AdminManager.scala

```diff
@@ -266,7 +266,8 @@ class AdminManager(val config: KafkaConfig,
     val allBrokerIds = allBrokers.map(_.id)
 
     // 1. map over topics creating assignment and calling AdminUtils
-    val metadata = newPartitions.map { case (topic, newPartition) =>
+    val metadata = newPartitions.map { newPartition =>
+      val topic = newPartition.name()
```

Comment:
nit: we typically drop parenthesis for simple accessors

Reply:
Removed. There were a bunch of other places where this was the case, got rid of them too.